src/EnergyPlus/DaylightingManager.cc

```diff
@@ -8620,25 +8520,25 @@ void ComplexFenestrationLuminances(EnergyPlusData &state,
             // Sun reaches ground point if vector from this point to the sun is unobstructed
             hitObs = false;
             for (int ObsSurfNum : state.dataSurface->AllShadowPossObstrSurfaceList) {
-                if (CalledFrom == DataDaylighting::CalledFor::RefPoint) {
-                    ComplexFenestrationLuminancesGroundHitPt(1) =
+                if (CalledFrom == CalledFor::RefPoint) {
+                    ComplexFenestrationLuminancesGroundHitPt.x =
                         state.dataBSDFWindow->ComplexWind(IWin).DaylghtGeom(CurCplxFenState).RefPoint(iRefPoint).GndPt(iGndElem, WinEl).x;
-                    ComplexFenestrationLuminancesGroundHitPt(2) =
+                    ComplexFenestrationLuminancesGroundHitPt.y =
                         state.dataBSDFWindow->ComplexWind(IWin).DaylghtGeom(CurCplxFenState).RefPoint(iRefPoint).GndPt(iGndElem, WinEl).y;
-                    ComplexFenestrationLuminancesGroundHitPt(3) =
+                    ComplexFenestrationLuminancesGroundHitPt.z =
                         state.dataBSDFWindow->ComplexWind(IWin).DaylghtGeom(CurCplxFenState).RefPoint(iRefPoint).GndPt(iGndElem, WinEl).z;
                 } else {
-                    ComplexFenestrationLuminancesGroundHitPt(1) = state.dataBSDFWindow->ComplexWind(IWin)
+                    ComplexFenestrationLuminancesGroundHitPt.x = state.dataBSDFWindow->ComplexWind(IWin)
                                                                       .DaylghtGeom(CurCplxFenState)
                                                                       .IlluminanceMap(iRefPoint, MapNum)
                                                                       .GndPt(iGndElem, WinEl)
                                                                       .x;
-                    ComplexFenestrationLuminancesGroundHitPt(2) = state.dataBSDFWindow->ComplexWind(IWin)
+                    ComplexFenestrationLuminancesGroundHitPt.y = state.dataBSDFWindow->ComplexWind(IWin)
                                                                       .DaylghtGeom(CurCplxFenState)
                                                                       .IlluminanceMap(iRefPoint, MapNum)
                                                                       .GndPt(iGndElem, WinEl)
                                                                       .y;
-                    ComplexFenestrationLuminancesGroundHitPt(3) = state.dataBSDFWindow->ComplexWind(IWin)
+                    ComplexFenestrationLuminancesGroundHitPt.z = state.dataBSDFWindow->ComplexWind(IWin)
                                                                       .DaylghtGeom(CurCplxFenState)
                                                                       .IlluminanceMap(iRefPoint, MapNum)
                                                                       .GndPt(iGndElem, WinEl)
```

Comment:
Similar comment as I had above:
ComplexFenestrationLuminancesGroundHitPt = state.dataBSDFWindow->ComplexWind(IWin)
                                                                      .DaylghtGeom(CurCplxFenState)
                                                                      .IlluminanceMap(iRefPoint, MapNum)
                                                                      .GndPt(iGndElem, WinEl)
Which would copy x,y and z. Not sure why is GndPt defined as Array2D<Vector> where using Vector = ObjexxFCL::Vector3<Real64>;

Reply:
`GndPt` is defined as an `Array2D` of `Vector` (which I think is the same as `Vector3<Real64>`). Don't look at me, man. I didn't write this code.